test/controllers/appointmentsController.spec.ts

```diff
@@ -369,5 +383,115 @@ describe('appointmentsController', () => {
           return Promise.resolve();
         });
     });
+
+    it('should not create appointment if diff in month > 4', async () => {
+      const psy = await clean.insertOnePsy();
+      const patient = await dbPatients.insert(
+        'Ada',
+        'Lovelace',
+        '12345678901',
+        '42',
+        false,
+        false,
+        psy.dossierNumber,
+        'Dr Docteur',
+        'adresse du docteur',
+        dateOfBirth,
+      );
+
+      const todayDate = new Date();
+      const newDatePlus5month = new Date(todayDate.setMonth(todayDate.getMonth() + 5));
+
+      return chai.request(app)
+        .post('/api/appointments')
+        .set('Cookie', `token=${cookie.getJwtTokenForUser(psy.dossierNumber, 'randomXSRFToken')}`)
+        .set('xsrf-token', 'randomXSRFToken')
+        .send({
+          patientId: patient.id,
+          date: newDatePlus5month,
+        })
+        .then(async (res) => {
+          res.status.should.equal(400);
+          res.body.message.should.equal('La date de la séance doit être dans moins de 4 mois');
+
+          const appointmentArray = await dbAppointments.getAll(psy.dossierNumber);
+          expect(appointmentArray).to.have.length(0);
+
+          return Promise.resolve();
+        });
+    });
+
+    it('should create appointment if date between 21/03/21 and today', async () => {
```

Comment:
what's the difference with the test should create appointment

Reply:
It was testing a date between the beginningDate and the limit of 4 month before today.
In case the 4 month limitation was blocking.